roles/edpm_nftables/tasks/service-bootstrap.yml

```diff
@@ -17,6 +17,25 @@
 - name: Gather service facts
   ansible.builtin.service_facts:
 
+- name: Check if firewalld service exists
+  ansible.builtin.command: systemctl status firewalld
+  register: firewalld_status
+  ignore_errors: true
+
+- name: Ensure firewalld service is disabled, stopped, and masked if it exists
+  ansible.builtin.systemd:
+    name: firewalld
+    enabled: false
+    state: stopped
+    masked: true
+  when: firewalld_status.rc == 0
+  register: manage_firewalld
+
+- name: Ensure deployment fails if firewalld management fails
+  fail:
+    msg: "Failed to manage firewalld service."
+  when: manage_firewalld is failed
```

Comment:
This will continue to fail in hosts that doesn't have firewalld installed, as `systemctl status firewalld` returns a non-zero exit code for unknown services. If the previous comment is considered this task won't be needed.

Reply:
done